planning/2021-q1.md

```diff
@@ -77,4 +79,17 @@ Nothing specific in Q1
 - [Tutorial: advanced MeiliSearch features](https://github.com/meilisearch/documentation/issues/708)
 - [Tutorial: intermediate MeiliSearch features](https://github.com/meilisearch/documentation/issues/707)
 - [Tutorial: pagination](https://github.com/meilisearch/devrel/issues/131)
-- [Tutorial: snapshot & dumps](https://github.com/meilisearch/devrel/issues/132)
\ No newline at end of file
+- [Tutorial: snapshot & dumps](https://github.com/meilisearch/devrel/issues/132)
+
+## Retrospective
+
+At the end of Q1, we took the time to reflect on the last three months. Here are the main topics of our discussion on the form of **O**bservations and possible **S**olutions on how to improve those points in Q2.
+
+**O:** We overestimated our team velocity and underestimated our day-to-day tasks (i.e.: Slack community, documentation PRs...) which results in failure to reach many of our Q1 goals. Our planning process was also too detailed for a startup at our stage.
+**S:** We are a new team, in a new company, so it was to be expected. We will focus on three main tasks per team member that must be achieved in the next quarter, and two tasks to tackle if the time permit.
+
+**O:** We often felt overwhelmed with everything we have to do to reach our goals, and to help the company move forward.
+**S:** It is a feeling we may have to live with for at least another quarter or two, until we have a stronger foundation and better processes when it comes to Developer Relations at MeiliSearch. We need to move from being in reaction mode to being more intentional with our time. One thing that will help is to better prioritize. To do so we will work with different prioritization levels for our tasks and focus only on the most important first. We will continue to use GitHub issues as a way to make a brain dump on things we need to do, even if they do not require immediate actions: things writing down freed some brain space. One additional problem is that we don't always everything else we do as accomplishment or "our work", meetings being a good example. It is a mindset to make ours.
```

Comment:
`One additional problem is that we don't always everything else we do as accomplishment or "our work", meetings being a good example.`  I don't understand this sentence, is it me or something is missing?

Reply:
Is it clearer now?

Reply:
Much clearer 😄